crates/re_query/src/util.rs

```diff
@@ -57,6 +57,11 @@ impl VisibleHistory {
         to: VisibleHistoryBoundary::Infinite,
     };
 
+    /// Returns the start boundary of the time range.
```

Comment:
```suggestion
    /// Returns the start boundary of the time range given an input cursor position.
```

Reply:
nit: I find the naming this function as "from" to be confusing.  Something like `range_start_from_cursor` would be much more clear.